src/project-management/project-management.ts

```diff
@@ -43,11 +43,11 @@ class ProjectManagementInternals implements FirebaseServiceInternalsInterface {
  * ProjectManagement service bound to the provided app.
  */
 export class ProjectManagement implements FirebaseServiceInterface {
+
   public readonly INTERNAL: ProjectManagementInternals = new ProjectManagementInternals();
 
-  private readonly resourceName: string;
-  private readonly projectId: string;
   private readonly requestHandler: ProjectManagementRequestHandler;
+  private projectId: string;
```

Comment:
It looks like this should be `private projectId?: string;` now

Reply:
Doesn't seem to be required for string and other primitive types.